form.blade.php

```diff
@@ -0,0 +1,204 @@
+@extends('layouts.main')
+@section('form')
+<div class="accordion container" id="accordionExample">
+    <h3>Education Details</h3>
+    {{-- 10th accordion --}}
+    <div class="">
+    <div class="accordion-item">
+      <h2 class="accordion-header" id="headingone_ssc">
+        <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+          10th
+        </button>
+      </h2>
+      <div id="collapseOne_ssc" class="accordion-collapse collapse show" aria-labelledby="headingOne" data-bs-parent="#accordionExample">
+        <div class="accordion-body">
+            <form class="row g-3">
+                <div class="col-md-12">
+                  <label for="inputCollege" class="form-label">School</label>
+                  <input type="text" class="form-control" id="inputCollege_ssc" name="school">
+                </div>
+                <div class="col-md-12">
+                  <label for="inputUniversity" class="form-label">State Board</label>
+                  <input type="text" class="form-control" id="inputUniversity_ssc" name="sscBoard">
```

Comment:
don't use camelCase use underscores to separate words. This is standard practice is php development
eg name="sscBoard" change to name="ssc_board"

Reply:
i have changed name to ssc_board